test/e2e_node/jenkins/cri_validation/benchmark-config.yaml

```diff
@@ -1,56 +1,56 @@
 ---
 images:
   gci-density1:
-    image: gci-dev-55-8872-18-0
+    image: gci-dev-56-8977-0-0
```

Comment:
Compared to `gci-dev-55-8872-18-0`, the change-log also includes several other things:
* Fixed a bug in e2fsprogs that caused mke2fs to take a very long time (Upstream fix: http://git.kernel.org/cgit/fs/ext2/e2fsprogs.git/commit/?h=next&id=d33e690fe7a6cbeb51349d9f2c7fb16a6ebec9c2)
* Enabled various kernel modules needed for iptables and conntrack functionality to work correctly
* Updated docker image used by toolbox. It now comes with several networking tools pre-installed